template/config/storybook/components/preview.vue

```diff
@@ -0,0 +1,152 @@
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import Example from "./example.vue";
+import Playground from './playground.vue';
+import Props from './props.vue';
+import { StoryComponent } from '.storybook/modules/story';
+
+const safe = new WeakMap<object, PreviewData>();
+
+interface PreviewData {
+  title?: string;
+  description?: string;
+}
+
+interface VueLike {
+  options: { name: string };
+}
+
+@StoryComponent({
+  name: "Preview",
+  components: {
+    Props,
+    Playground
+  }
+})
+export default class Preview extends Vue {
+  // --------------------------------------------------------------------------
+  // Fields
+  // --------------------------------------------------------------------------
+
+  @Prop()
+  public for: VueLike | undefined;
+
+  private component: VueLike | undefined = undefined;
+  private description: string | undefined = "";
+  private importStatement = "";
+  private title = "";
+  private activeTab = 0;
+  private enablePlayground = true;
+  private enableApi = true;
+
+  // --------------------------------------------------------------------------
+  // Accessors
+  // --------------------------------------------------------------------------
+
+  public get importStatementPretty() {
+    return (this.importStatement ?? "").replace(
+      /import|from/g,
+      p => `<span class="text-red-400 font-bold">${p}</span>`
+    );
+  }
+
+  // --------------------------------------------------------------------------
+  // Methods
+  // --------------------------------------------------------------------------
+
+  private mounted() {
+    const component = this.resolveStoryComponent() ?? {};
+    const componentModule = StoryComponent.getModule(component);
+
+    this.component = component;
+    this.title = component.options?.name;
+    this.description = StoryComponent.getDescription(component);
+    this.enablePlayground = StoryComponent.getPlaygroundEnabled(component);
+    this.enableApi = StoryComponent.getApiEnabled(component);
+
+    if (componentModule !== undefined) {
+      const componentModuleStr =
+        typeof componentModule === "string"
+          ? (componentModule as string)
+          : componentModule(this.title);
+
+      this.importStatement = `import ${this.title} from '${componentModuleStr}';`;
+    }
+  }
+
+  private resolveStoryComponent() {
+    let component: VueLike | undefined;
+
+    if (this.for === undefined) {
+      component = Object.values(
+        this.$parent.$options.components ?? {}
+      ).find(p => StoryComponent.isDefined(p)) as VueLike;
+    } else {
+      component = this.for;
+    }
+
+    // If (component === undefined) {
```

Comment:
Should this be commented out? @taslangraham

Reply:
@taslangraham still seeing the commented out code, should this be here or should it be removed?